src/SourceBuild/content/Directory.Build.targets

```diff
@@ -8,7 +8,7 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <SdkTarballItem Condition="'$(SdkTarballPath)' != ''" Include="$(SdkTarballPath)" />
+      <SdkTarballItem Condition="'$(SdkTarballPath)' != ''" Include="$(SdkTarballPath) and '$(SmokeTestsLicenseScanPath)' == ''" />
```

Comment:
From a separation of concerns POV this doesn't look right. This target shouldn't know about tests or test properties. Can you please describe the overall issue in more detail?